src/rsa/padding.rs

```diff
@@ -154,3 +155,150 @@ impl Verification for PKCS1 {
         })
     }
 }
+
+/// PSS Padding as described in https://tools.ietf.org/html/rfc3447#section-9.1.
+/// It generates a random salt equal in length to the output of the specified
+/// digest algorithm and uses MGF1 as the mask generating function.
+pub struct PSS {
+    digest_alg: &'static digest::Algorithm,
+}
+
+// The maxmimum supported output length for PSS padding is equal to the maximum
+// supported RSA modulus length.
+// TODO: Can we avoid requiring this MAX_OUTPUT_LEN buffer?
+const MAX_OUTPUT_LEN: usize = 8192 / 8;
```

Comment:
We can drop the TODO because we already have an issue filed for removing the buffer.